tsconfig.json

```diff
@@ -15,5 +15,6 @@
     "noImplicitAny": false,
     "skipLibCheck": true
   },
-  "exclude": ["node_modules", "dist", "media", "**/*.spec.ts", "tests"]
```

Comment:
@jmcdo29 

To resolve, the following error message when I hit `npm run commit` command while committing the test code, I've followed `nestjs-ogma` configuration by adding `tsconfig.build.json` file and it seems working now.

```bash
❌ tslint -p tsconfig.json

~/nestjs-multer-extended/lib/multer-sharp/multer-sharp.utils.spec.ts' is not included in project.
```

Reply:
I'd say this will be a more scalable and manageable solution having separate ts and jest configs for tests and build. It'll definitely make it easier to run everything when it is needed.